core/src/replay_stage.rs

```diff
@@ -1728,6 +1732,7 @@ impl ReplayStage {
         voting_sender: &Sender<VoteOp>,
         epoch_slots_frozen_slots: &mut EpochSlotsFrozenSlots,
         bank_drop_sender: &Sender<Vec<Arc<Bank>>>,
+        wait_to_vote_slot: &Option<Slot>,
```

Comment:
Likely `&` is not useful here. `Option<Slot>` implements `Copy`. 
https://www.forrestthewoods.com/blog/should-small-rust-structs-be-passed-by-copy-or-by-borrow/